app/Data/Repositories/InsightsRepository.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Data\Repositories;
+
+use App\Data\Collections\MediaCollection;
+use App\Data\Models\Account;
+use App\Data\Models\Metrics;
+use Illuminate\Support\Facades\Redis;
+
+class InsightsRepository
+{
+    /**
+     * @param Account $account
+     * @param string $platform
+     * @return bool
+     */
+    public function storeAccount(string $platform, Account $account): bool
+    {
+        $key = "insights:{$platform}:{$account->username}:latest:account";
+        return Redis::set($key, json_encode([
+            "following" => $account->following,
```

Comment:
why not call `$account->toArray()` instead of re-creating the same array structure. Also applies to storing media metrics `storeMediasMetrics`